settings.py

```diff
@@ -1254,8 +1253,10 @@
     }
 }
 
-LOGGING['handlers'].update(LOCAL_LOGGING_HANDLERS)
-LOGGING['loggers'].update(LOCAL_LOGGING_LOGGERS)
+if LOCAL_LOGGING_CONFIG:
+    for key, config in LOCAL_LOGGING_CONFIG.items():
+        if key in ('handlers', 'loggers', 'formatters', 'filters'):
```

Comment:
I think keys in `LOCAL_LOGGING_CONFIG` that are not in this tuple will be silently ignored because of this conditional. Would it make sense to raise an error for other keys? Alternately, maybe it would make more sense to remove the conditional? Are there keys that should be excluded?

Reply:
I could raise an error. The reason not to allow ALL keys is that not all of them are dictionaries so either we need to type check or we just let the error get raised if we call '.update' on a string. I thought this was 'safer' but maybe the cost isn't worth it?

Reply:
> maybe the cost isn't worth it?

That's my feeling. Better to raise an error (or implicitly let one be raised for non-dict value) than to silently ignore some keys.

Reply:
How about this https://github.com/dimagi/commcare-hq/pull/25847